recipes/oss

```diff
@@ -0,0 +1,27 @@
+name=oss
+version=4.2.2020
+revision=1
+tarball_url="http://www.4front-tech.com/developer/sources/stable/gpl/oss-v4.2-build2020-src-gpl.tar.bz2"
+tarball_blake2b="739aabd8b1851a3878d62b101e91552b647b2b402c83e539d1150c52809b72efbe3c7a93f08ba6b57132f5c20763345fb9579f79315b7a137faec9135a664652"
+source_allow_network=yes
+source_imagedeps="curl"
+imagedeps="gcc"
```

Comment:
If this imagedep isn't needed, remove it as well.

Reply:
The imagedep is actually needed as it builds some tools that it runs during the build process (originally those were built using $CC but because that doesn't work with cross-compilation the patch changes those to use cc).